lib/grinder_sdk.dart

```diff
@@ -33,6 +33,51 @@ Directory getSdkDir([List<String> cliArgs]) => cli_util.getSdkDir(cliArgs);
 
 File get dartVM => joinFile(sdkDir, ['bin', _sdkBin('dart')]);
 
+/// Utility tasks for for getting information about the Dart SDK and invoking
+/// dart applications.
+class Dart {
```

Comment:
I left this as `Dart`, and removed the `location` static field. That would apply to a wrapper around the dartsdk, but not one around the dartvm.